tdms/include/interpolation_methods.h

```diff
@@ -24,64 +24,66 @@
  *    o    o    o    o    o    o    o    o
  * We label these interp positions with the index of the point to the left: a data point at interp position 0 is effectively a data point "v0.5", being at the midpoint of v0 and v1, for example.
  */
-enum scheme_value
-{
-    BAND_LIMITED_0 = 4,             // use bandlimited interpolation w/ interp position = 0
-    BAND_LIMITED_1 = 6,             // use bandlimited interpolation w/ interp position = 1
-    BAND_LIMITED_2 = 8,             // use bandlimited interpolation w/ interp position = 2
-    BAND_LIMITED_3 = 9,             // use bandlimited interpolation w/ interp position = 3 [Preferred method if available]
-    BAND_LIMITED_4 = 7,             // use bandlimited interpolation w/ interp position = 4
-    BAND_LIMITED_5 = 5,             // use bandlimited interpolation w/ interp position = 5
-    BAND_LIMITED_6 = 3,             // use bandlimited interpolation w/ interp position = 6
-    BAND_LIMITED_7 = 2,             // use bandlimited interpolation w/ interp position = 7
-    BAND_LIMITED_CELL_ZERO = 1,     // use bandlimited interpolation to interpolate to the centre of Yee cell 0
-    CUBIC_INTERP_MIDDLE = 0,        // cubic interpolation to middle 2 of 4 points (interp1)
-    CUBIC_INTERP_FIRST = -1,        // cubic interpolation to first 2 of 4 points (interp2)
-    CUBIC_INTERP_LAST = -2          // cubic interpolation to last 2 of 4 points (interp3)
+enum scheme_value {
+  BAND_LIMITED_0 = 4,// use bandlimited interpolation w/ interp position = 0
+  BAND_LIMITED_1 = 6,// use bandlimited interpolation w/ interp position = 1
+  BAND_LIMITED_2 = 8,// use bandlimited interpolation w/ interp position = 2
+  BAND_LIMITED_3 =
+          9,// use bandlimited interpolation w/ interp position = 3 [Preferred method if available]
+  BAND_LIMITED_4 = 7,// use bandlimited interpolation w/ interp position = 4
+  BAND_LIMITED_5 = 5,// use bandlimited interpolation w/ interp position = 5
+  BAND_LIMITED_6 = 3,// use bandlimited interpolation w/ interp position = 6
+  BAND_LIMITED_7 = 2,// use bandlimited interpolation w/ interp position = 7
+  BAND_LIMITED_CELL_ZERO =
+          1,// use bandlimited interpolation to interpolate to the centre of Yee cell 0
```

Comment:
(Soft opinion) I want to place it above rather than split the line. Should play well with doxygen too.